common/auxip-selection/src/main/java/com/csgroup/s2pdgs/auxip/S2AuxList.java

```diff
@@ -66,14 +66,14 @@ public S2AuxList(String catalogIndex, List<AuxType> auxTypes, Catalog catalog,
         }
         LOGGER.info("Searching for dates : %s + %s to %s + %s".formatted(start.toString(),dt0.toString(),
                 end.toString(), dt1.toString()));
-        for (AuxType t : auxList) {
-            LOGGER.info("Treating : " + t.getName());
+        for (AuxType t : auxList) {            
             String prefix = "s2_*";
             if (t.isUnitDependent()) {
                 prefix = ("S2" + unit.name() + "_*").toLowerCase();
             }
             for (String s : t.getProductLevels()) {
                 if (s.equals(level.name())) {
+                    LOGGER.info("Treating : " + t.getName());
```

Comment:
Avoid concatenation in log, use templating

```suggestion
                    LOGGER.info("Treating : {}" , t.getName());
```